tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -151,6 +157,27 @@ def test_superuser_can_delete_other_user(self):
         self.assertRedirects(response, "/admin/auth/user/")
         self.assertFalse(get_user_model().objects.filter(pk=self.tester.pk).exists())
 
+    def test_superuser_cannot_delete_the_last_superuser(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.admin.username, password="admin-password"
+        )
+        self.client.get(reverse("admin:auth_user_delete", args=[self.admin2.pk]))
+        self.client.post(
+            reverse("admin:auth_user_delete", args=[self.admin2.pk]),
+            {"post": "yes"},
+            follow=True,
+        )
+        response = self.client.get(
```

Comment:
Add a comment above just for reference to indicate that there's only 1 super-user left. You can also assert that the number of super-users is 1 on the line above and at least 2 before deleting `admin2`.